docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,68 @@
+# Release 450 (7 Jun 2024)
+
+## General
+
+* Add support for the `optimizer.push-filter-into-values-max-row-count`
+  configuration property, which can define a number of rows to be pushed from a
+  filter into values. ({issue}`22105`)
+* Improve performance of the {func}`first_value` and {func}`last_value`
+  functions. ({issue}`22092`)
+* Improve performance for large clusters under heavy workloads. ({issue}`22039`)
```

Comment:
super vague ..

Reply:
I'm not really sure how else to word it...

Reply:
yeah .. fine to leave as is